src/optimizer/join_order/join_order_optimizer.cpp

```diff
@@ -96,8 +96,9 @@ static unique_ptr<LogicalOperator> PushFilter(unique_ptr<LogicalOperator> node,
 bool JoinOrderOptimizer::ExtractJoinRelations(LogicalOperator &input_op, vector<LogicalOperator *> &filter_operators,
                                               LogicalOperator *parent) {
 	LogicalOperator *op = &input_op;
-	while (op->children.size() == 1 && (op->type != LogicalOperatorType::LOGICAL_PROJECTION &&
-	                                    op->type != LogicalOperatorType::LOGICAL_EXPRESSION_GET)) {
+	while (op->children.size() == 1 &&
```

Comment:
Perhaps it makes sense to use a different operator type for table in-out functions, instead of also using a `LogicalGet`? @Tishj

Reply:
@Mytherin The issue exists on master, and I think my PRs for the scalar table function are too big to also include this.
I think we should merge this as-is and I'll change it with a different operator type for the table in-out functions

Reply:
Yes, that is fair